Makefile

```diff
@@ -7,9 +7,15 @@ all: test
 	@pip install -e .
 	@touch .develop
 
+
+flake: .develop
+	flake8 yarl
```

Comment:
I think this should be run as a part of `make test` to simplify development process. And no need to intorduce another target since pytest uses flake8 and can run it against regular sources.

Reply:
mmm...by looking on `.travis.yml` and `Makefile` it seems that there is need to make some refactoring to achieve that.

 @asvetlov, what do you think?

Reply:
During the tests run we check flake errors only at tests folder:
```
test: .develop
	pytest ./tests --flake8
```
Maybe it would be better:
```
test: .develop
	flake8 yarl tests
	pytest ./tests
```

Reply:
I think it would be fine to just run `pytest --flake8` - it will run both tests and flake8 on everything in directory: yarl, setup.py, tests and the rest bits. That shouldn't hurt. If you use `pytest-sugar`, the result report will be pretty nice.

Reply:
I did not notice your message =(